conformance-test/libraryMethods.ts

```diff
@@ -21,6 +21,7 @@ import {
   deleteTestFile,
 } from './testBenchUtil';
 import * as uuid from 'uuid';
+import {getDirName} from '../src/util.js';
```

Comment:
NB, you could save dirName = getDirName(), that way you improve performance vs. memory (seems like you call it in multiple spots)